config/credentials.yml.enc

```diff
@@ -1 +1 @@
-PFpsVGTgu+qEO5lIxLd3qRjRrLWoTeRZUHlwl6senVyKAVjPtQha/dLG9W6ZayqCV4gm+xaOyBwV7e6TnMvCHbTCkHwtXoZvohlofi259V8UfFQ9LOl26VuolvlV04sY7U0Bn7cl+ncIlpvCFPOjmEQtzKJfJ1FF6UqjNJqxwUWXhNYVmdW31XloGQXo9JRUGwPb1rEXhBT/UDBfEWSnXASSlroqmlusbe1XGTfeRZO3XumTP36tEH673EleWyCBcp7wkCx2mfFvCOo+rKceqdOm7SfIQiWv8KHWrcr28SPzFwWiY/2vDEGp+vI8YVT5aiY4cJMnJdMd9FL9oA2x6+1HhTzfoM1mi+AMwWrDFl9flGeo0y/Xncmc9L0cMX8KPqYyVhueNby/xHqdzj7BtGUOAwBt--4AOWp9W4t0pynmaL--YEaZ3SY4sWEFiEWilFwhJQ==
\ No newline at end of file
+SjNvOjdocFGawdWGz00OJU3yla+1Up1L5t04BFnYne3jZGA1GDXo3Xw46/59AKLyPLcX0wwx3RBvzH8Yv8Zt/EcfP8rXHsoKmp9Tv45hZmpbpyvxMwPP5gBdEIQ+iQQnOB4rpSxI52p1+1UVAeKmP1l5CDo3yU3EO/DPGsDx+Ksi53zcVPIRGhRdPg8qlx+xGXtcHkiTltpfwRA5aVhsSXFCwsoVXBf3w3HF+gDRytfrPcPlsdNvcbPCifW/U27M6flBuqeeAl3vPkJUruwbw4+2MwVWEXP7fWoGRstCbdN1TuxRd87bg911n7I4P8f63I+lLZTop5ar6evOgy74xE6Txv5DCLFirz0op2V00DO1v0vm1V7Fa4VVzzd/lMzgJaFz6Xa1ddJ4veyqeQIvSzUsKQ/o--OlMO+L6asi/XnYul--I2s6LF8u42K8+H5C9zgXzg==
```

Comment:
What did you change? I'm not able to decrypt this with the master key that you sent to Ron.

Reply:
@dsinn https://github.com/ElenaCherpakova/RoR-practicum/pull/76 should resolve this no?

Reply:
Yeah, unless you needed to add anything to the credentials file, which would be inconsistent if you're using dotenv, this file should not appear in this PR.